### DISCUSSION TITLE
Share multiple albums at once

### DISCUSSION THREAD

**Discussion by ph00lt0** [ente-io/ente#902] [Enhancements]
I have a looot of albums (going back many years) I wish to share with family members. It's quite impossible to share them all hand by hand with the family members. It would be great If I can just select many of them and share them at once.

**Comment by JanOstrowka** [comment]
I've been facing the same issue, struggling to share many albums in one go.
Most of all, it's hard to browse all the albums on desktop, without dedicated album view (like on mobile)

**Reply by cykl** [reply]
>  Most of all, it's hard to browse all the albums on desktop, without dedicated album view (like on mobile)

Even on mobile, the grid view make it difficult to browse through many collections. Collection names are often truncated and I don't really care about thumbnails.

It would be great to be able to display one collection per line with a smaller thumbnail.

**Comment by ph00lt0** [comment]
@ua741 hey man, could you explain why is this blocked now? 
I am really hoping to get this feature. 

Also, to add to this, it would be nice if you could add multiple people at once too. Right now I not only need to share per album but also per user. That's a lot of work for going through all the memories with the same friends / family.

**Comment by vishnukvmd** [comment]
Hey, this is blocked on visuals for the selected state and an action bar that displays available actions for the selected albums.

However, we don't need new designs to share a single album with multiple contacts. Have created an issue here: https://github.com/ente-io/ente/issues/823

**Reply by ph00lt0** [reply]
any updates on this?

**Comment by dermitdemdino** [comment]
A "must have" Feature: Share all with one click

**Comment by chiraag-nataraj** [comment]
Not sure how this works when each album has its own URL (generated client-side, I believe). @vishnukvmd, could probably iterate through all selected albums, generate URLs, and list them all I guess?
